### PR TITLE
[docs] Mention utf-8 encoding specifics in README

### PR DESCRIPTION
**Pull request by olivielpeau**
### What does this PR do?

Clarifies plugin's documentation with relation to expected logs' encoding and how to convert it if needed.

### Motivation

User report.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +122,24 @@ Configuration example:
...
+### Encoding
+
+Datadog's API expects log messages to be encoded in UTF-8.
+If some of your logs are encoded with a different encoding, we recommend using the [`record_modifier` filter plugin](https://github.com/repeatedly/fluent-plugin-record-modifier#char_encoding)
+to encode these logs to UTF-8.
+
+Configuration example:
+
+```
+# Change encoding of logs tagged with "datadog.**"
```
Comment (ogaca-dd) [inline README.md:134]: Are the 2 stars `**` expected?

File: README.md [hunk 2]
```diff
@@ -123,6 +122,24 @@ Configuration example:
...
+<filter datadog.**>
```
Comment (ogaca-dd) [inline README.md:135]: If this line is correct, then line 120 is probably wrong as the number of `*` is not the same.

**Review by ogaca-dd — APPROVED** [review]
LGTM.

File: README.md (hunk 1 above)
Comment (olivielpeau) [reply README.md:134]: Yes, this was confirmed to work

File: README.md (hunk 2 above)
Comment (olivielpeau) [reply README.md:135]: I'm not entirely sure about line 120, I don't have a test setup to confirm right now, I'll just go ahead and merge for now

**Pull request merged**